build.cake

```diff
@@ -0,0 +1,243 @@
+// Usings
+using System.Xml;
+using System.Xml.Linq;
+using System.Xml.XPath;
+
+// Arguments
+var target = Argument<string>("target", "Default");
+var source = Argument<string>("source", null);
+var apiKey = Argument<string>("apikey", null);
+var version = Argument<string>("targetversion", null);
+var incremental = Argument<bool>("incremental", false);

```

Comment:
Yes. Should probably be called "skipclean" or something like that. This is just something I use myself at the moment.